### PR TITLE
Added example ShowScreenResolution

### PR DESCRIPTION
**Pull request by kolibril13**

### CODE REVIEW

File: docs/source/examples/shapes_images_positions.rst [hunk 1]
```diff
@@ -1,5 +1,22 @@
 Shapes, Images and Positions
 =================================
+.. manim:: ShowScreenResolution
+    :save_last_frame:
+
+    class ShowScreenResolution(Scene):
+        def construct(self):
+            pyFrame = config["pixel_height"]  # 1080 default
+            pxFrame = config["pixel_width"]  # 1920 #default
```
Comment (leotrs) [inline docs/source/examples/shapes_images_positions.rst:9]: Nitpick (non-blocking): I found these variable names confusing: at first I thought they were both named pxFrame. At first I thought "px" stood for pixel and "py" stood for python. Also, these names do not technically adhere to PEP8 recommendations (snake_case is preferred for variable names over camelCase).

**Review by leotrs — COMMENTED** [review]
Out of curiosity, could you post a picture of the final output please?

File: docs/source/examples/shapes_images_positions.rst (hunk 1 above)
Comment (kolibril13) [reply docs/source/examples/shapes_images_positions.rst]: ok, changed it

**Comment by kolibril13** [comment]
> Out of curiosity, could you post a picture of the final output please?

ShowScreenResolution

**Comment by leotrs** [comment]
Thanks. This looks good to me provisionally, but in the future I'd move this example to be part of some tutorial.

**Review by leotrs — APPROVED** [review]

**Review by behackl — APPROVED** [review]
I really like the example, but I don't like the place you put it to. I agree with @leotrs that it should be part of a tutorial.

On a related note, I'd really like to reorganize the Example Gallery before our next release in order to implement the stricter separation between it an the reference manual.

**Comment by kolibril13** [comment]
Totally agree on that!
I think that reorganizing things in prs is not the best way to incrementally make small changes, which we will do when it comes to sorting between reference and gallery, and I think this is not a task one person will do alone.
So I suggest, we make a new branch called "docs-sorting" were we just push our changes without the necessity of reviews and when this is ready, we make a pr to the master.

**Pull request merged**